mmengine/runner/runner.py

```diff
@@ -1647,6 +1661,14 @@ def val(self) -> dict:
         Returns:
             dict: A dict of metrics on validation set.
         """
+        if is_model_wrapper(self.model):
+            ori_model = self.model.module
+        else:
+            ori_model = self.model
+        assert hasattr(ori_model, 'val_step'), (
+            'If you want to train your model, please make sure your model '
+            'has implemented `val_step`.')
```

Comment:
This checking is not required because `runner.val` will call `self.val_loop.run` soon.